runelite-client/src/main/java/net/runelite/client/plugins/fps/FpsConfig.java

```diff
@@ -29,9 +29,9 @@
 import net.runelite.client.config.ConfigItem;
 
 @ConfigGroup(
-	keyName = FpsPlugin.CONFIG_GROUP_KEY,
-	name = "FPS Control",
-	description = "Lets you control what your game frame rate is, often helps keep CPU down too"
+		keyName = FpsPlugin.CONFIG_GROUP_KEY,
```

Comment:
What is with these extra spaces?

Reply:
A bunch of the config stuff was double indented like that and I tried to fix it, I guess I missed some. Mb

Reply:
Fixed